vms/job/tests/test_jobDetails.py

```diff
@@ -11,53 +10,183 @@
 from shift.utils import (create_admin, create_event_with_details,
                          create_job_with_details)
 
-# Class contains failing test cases which have been documented
-# Test class commented out to prevent travis build failure
-"""
 
 class JobDetails(LiveServerTestCase):
-    '''
-    Contains Tests for View Job Details Page
-    '''
+    """
+    Contains Tests for Job app.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        cls.driver = webdriver.Firefox()
+        cls.driver.implicitly_wait(5)
+        cls.driver.maximize_window()
+        cls.job_details_page = JobDetailsPage(cls.driver)
+        cls.authentication_page = AuthenticationPage(cls.driver)
+        super(JobDetails, cls).setUpClass()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.driver.quit()
+        super(JobDetails, cls).tearDownClass()
 
     def setUp(self):
         create_admin()
-        self.job_list_page = '/job/list/'
-
-        self.driver = webdriver.Firefox()
-        self.driver.implicitly_wait(5)
-        self.driver.maximize_window()
-        super(JobDetails, self).setUp()
-        self.job_details_page = JobDetailsPage(self.driver)
-        self.authentication_page = AuthenticationPage(self.driver)
+        self.login_admin()
 
     def tearDown(self):
-        self.driver.quit()
-        super(JobDetails, self).tearDown()
-
-    def register_job(self):
-        # create shift and log hours
-        created_event = create_event_with_details(
-            ['event', '2017-06-15', '2017-06-17'])
-        created_job = create_job_with_details(
-            ['job', '2017-06-15', '2017-06-18', '', created_event])
+        pass
```

Comment:
Try logging out for every test otherwise logout 1 time in class tear down.

Reply:
@naman1901 @pavankm I have a doubt regarding this.
If we logout only once then build time won't be affected but if we logout each time i.e. in `tearDown` the build time will increase since it will logout for each test.

On the other hand the if we logout once that is less correct than each time, since each test is considered independent of the other and should end their session at end of test.

Need your views on which approach to go with.

Reply:
Umm not really but I did notice ~5 minutes minimum increase in boot time although the bu8ld time doesn't really increase by more than ~1 minute

Reply:
Ideally, we should follow the correct approach even if it takes more time. Waiting for @pavankm to present his views.